surveys/v_surveys.manager_survey_detail.sql

```diff
@@ -32,12 +32,12 @@ SELECT d.survey_id
       ,d.subject_samaccountname
       ,d.subject_manager_name
       ,d.subject_manager_samaccountname
-      ,w.job_name AS subject_dayforce_role
+      ,w.job_title AS subject_dayforce_role --should change column name alias
 FROM gabby.surveygizmo.survey_detail d
-LEFT JOIN gabby.dayforce.employee_work_assignment w
-  ON d.subject_df_employee_number = w.employee_reference_code
- AND d.date_submitted BETWEEN w.work_assignment_effective_start AND w.work_assignment_effective_end
- AND w.primary_work_assignment = 1
+LEFT JOIN gabby.people.employment_history w
+  ON d.subject_df_employee_number = w.employee_number
+ AND d.date_submitted BETWEEN w.effective_start_date AND w.effective_end_date
+ AND w.is_current_record = 1
```

Comment:
`is_current_record` could contradict the date join, you want to use only one or the other